supersuit/aec_wrappers.py

```diff
@@ -548,3 +550,64 @@ def __init__(self, env, lower_bound=-1, upper_bound=1):
 
     def _change_reward_fn(self, rew):
         return max(min(rew, self.upper_bound), self.lower_bound)
+
+class nan_noop_wrapper(BaseWrapper):
+    '''
+    this wrapper expects there to be a no_op_action parameter which
+    is the action to take in cases when nothing should be done.
+    '''
+    def step(self, action, no_op_action, observe = True):
+        if np.isnan(action).any():
+            warnings.warn("[WARNING]: Step received an NaN action {}. Evironment is {}. Taking no operation action.".format(action, self))
+            action = no_op_action
+        return super().step(action)
+
+class nan_zeros_wrapper(BaseWrapper):
+    '''
+    this wrapper warns and executes a zeros action when nothing should be done.
+    Only for Box action spaces.
+    '''
+    def step(self, action, observe=True):
```

Comment:
`observe=True` as an argument to `step` was removed from the API. So even though the old examples had it, it should not be there in supersuit now. Sorry I didn't mention this earlier